spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -259,6 +259,53 @@
       expect(log_item.admin_id).to eq(admin.id)
       expect(log_item.note).to eq("Change made by #{admin.login}")
     end
+
+    it "logs updating a fannish next of kin" do
+      admin = create(:support_admin)
```

Comment:
I notice there are repeated lines of 
```
      admin = create(:support_admin)
      fake_login_admin(admin)
```

We could surround the FNOK tests with a shared context to be more efficient.

```
    context "when admin has support role" do
      let(:admin) { create(:support_admin) }
```